traefik/Chart.yaml

```diff
@@ -1,6 +1,6 @@
 apiVersion: v1
 name: traefik
-version: 8.2.1
+version: 8.2.2
```

Comment:
As this PR is adding new configuration, it should rather be a minor enhancement then a patch version

Reply:
I updated the version to 8.5.0